.travis.yml

```diff
@@ -4,6 +4,7 @@ branches:
     - master
 language: ruby
 rvm:
-  - 2.6.6
-  - 2.5.8
-  - 2.4.10
+  - 3.2.1
+  - 3.1.3
+  - 3.0.0
+  - 2.7.7
```

Comment:
recent patch versions, and the most recent security patch for ruby 2. Not sure if it's worth leaving 3.0.0 in as well